src/NorthwindWebsite/API/Controllers/RootController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace NorthwindWebsite.API.Controllers;
+
+[Route("api")]
+[ApiController]
+public class RootController : ControllerBase
+{
+    public ActionResult GetRoot()
```

Comment:
I think you may also parametrise `ActionResult` with `object` to avoid extra `Ok` wrapping